.editorconfig

```diff
@@ -14,7 +14,6 @@ trim_trailing_whitespace = true
 
 [*.{yml,json}]
 indent_size = 2
-indent_style = space
```

Comment:
Why was this removed? We do want 2 spaces

Reply:
It's removed because the `indent_style` property with the same value is already defined in the `[*]` section so it's redundant and not applied.